frontend/views.py

```diff
@@ -76,3 +77,19 @@ def get(self, request, *args, **kwargs):
             "country": Country.objects.get(slug=self.kwargs.get("slug"))
         }
         return render(request, 'frontend/expenses-and-revenues.html', context=cn)
+
+
+class NewPage(View):
```

Comment:
Name your views meaningfully. 'NewPage' does not tells what this view represents.